src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkCorrectUsingMethodsInspection.kt

```diff
@@ -16,23 +23,25 @@ class SmkCorrectUsingMethodsInspection: SnakemakeInspection() {
             session: LocalInspectionToolSession
     ) = object : SnakemakeInspectionVisitor(holder, session) {
 
-        private fun checkAncient(node: PsiElement): PsiElement? {
-            if (node.text != "input")
-                return node
+        private fun checkAncient(psiEl: PsiElement): PsiElement? {
+            if (psiEl.text != SECTION_INPUT)
```

Comment:
The method is too generic, accepts any PSI element but you call it actually for SmkRuleOrCheckpointArgsSection

Reply:
Done